collectors/mimecast/test/mimecast_mock.js

```diff
@@ -27,29 +27,40 @@ const AIMS_TEST_CREDS = {
 };
 
 const LOG_EVENT = {
-    "result": "result",
-    "date": "2020-10-01T23:59:59+0000",
-    "senderAddress": "senderAddress",
-    "fileName": "fileName",
-    "actiontriggered": "actiontriggered",
-    "route": "route",
-    "details": "details",
-    "recipientAddress": "recipientAddress",
-    "fileType": "fileType"
+    "senderAddress":"senderAddress",
+    "recipientAddress":"recipientAddress",
+    "fileName":"Test CIS Report 2021-05-25 00:00 Test.pdf",
+    "fileType":"application/pdf",
+    "result":"safe",
+    "actionTriggered":"none",
+    "date":"2021-05-25T00:00:09+0000",
+    "details":"Safe Time taken: 0 hrs, 0 min, 4 sec",
+    "route":"inbound",
+    "messageId":"messageId@email.sssss.com",
```

Comment:
if `messageId` is an email then we don't want to use it as message type.

Reply:
it's looks like eee1234550eedsdedrfff23eee223eeaa@email.domain.com

Reply:
Which message type I use